developer_guide.md

```diff
@@ -0,0 +1,71 @@
+# Az NHC Developer Guide #
+
+## Pre-requisites  ##
+
+  - Running on a VM which has the environment in which you would like AzNHC to run in.
+  - Install AzNHC
+
+## Creating a custom NHC configure file for your environment ##
+
+1. You have two choices when creating a conf file.
+   - You can generate a conf file using nhc-gen (Recommended). The following steps cover this scenario.
+   - You can write one from scratch. Look at existing conf files for reference.
```

Comment:
Start this with "Or you can write..." just to be clear for users scanning documentation?

Reply:
addressed